graphics/qvge/Portfile

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+PortGroup           qmake5 1.0
+
+github.setup        ArsMasiuk qvge 0.6.3 v
+checksums           rmd160  fbcecf18b32976d11cb4949330b6cce66349acac \
+                    sha256  f08cbcae4bb4911c5564d929288b8c85f3994d9c0018c929ce37efce239dab7a \
+                    size    458064
+
+name                qvge
+version             0.6.3
```

Comment:
`name` and `version` are not required - you have already implicitly set them in your `github.setup` line.  As a result, these redundant `name` and `version` lines can be removed.

Reply:
Portfile is updated accordingly.